test/unit/mongo/MongoTripodConfigTest.php

```diff
@@ -56,30 +56,40 @@ public function testNamespaces()
     public function testTConfig()
     {
         $config = MongoTripodConfig::getInstance();
+        $cfg = MongoTripodConfig::getConfig();
         $tConfig = $config->getTransactionLogConfig();
-        $this->assertEquals('testing',$tConfig['database']);
+        $this->assertEquals('tripod_php_testing',$tConfig['database']);
         $this->assertEquals('transaction_log',$tConfig['collection']);
-        $this->assertEquals('mongodb://localhost',$config->getTransactionLogConnStr());
+        $this->assertEquals($cfg['data_sources'][$cfg['transaction_log']['data_source']]['connection'],$config->getTransactionLogConnStr());
     }
 
     public function testTConfigRepSetConnStr()
     {
         $config=array();
+        $config["data_sources"] = array(
```

Comment:
might be as simple as setting a default config in beforeEach and then overriding specific properties in actual test function before instantiating MongoTripodConfig.